helpers/spark-defaults.conf

```diff
@@ -0,0 +1,7 @@
+spark.app.name=canonical-spark-default-app
+spark.executor.instances=2
+spark.kubernetes.container.image=docker.io/averma32/sparkpy6:latest
+spark.kubernetes.container.image.pullPolicy=Always
```

Comment:
not sure about this. Always can be super annoying because it's usually slower, especially on stable base images that don't really change.